src/vsr/replica.zig

```diff
@@ -5672,6 +5667,14 @@ pub fn ReplicaType(
             if (self.status == .view_change and self.log_view < self.view) {
                 if (!self.do_view_change_quorum) self.send_do_view_change();
             }
+
+            if (self.solo()) {
+                if (self.commit_min < self.op) {
```

Comment:
Can we assert that we are `.recovering`? If we are solo, we can only update view durable on startup, correct?